cli/config/configfile/file.go

```diff
@@ -351,3 +351,26 @@ func (configFile *ConfigFile) SetPluginConfig(pluginname, option, value string)
 		delete(configFile.Plugins, pluginname)
 	}
 }
+
+// GetFeature retrieves the given key from the config features map.
+func (configFile *ConfigFile) GetFeature(key string) (string, bool) {
+	if configFile.Features == nil {
+		return "", false
+	}
+	v, ok := configFile.Features[key]
+	return v, ok
+}
+
+// SetFeature sets the key to the given value in the config features map.
+// If the features field is nil, it initializes a new map.
+// Passing a value of "" will remove the key-value pair from the map.
+func (configFile *ConfigFile) SetFeature(key, value string) {
+	if configFile.Features == nil {
+		configFile.Features = make(map[string]string)
+	}
+	if value != "" {
+		configFile.Features[key] = value
+	} else {
+		delete(configFile.Features, key)
```

Comment:
nit: if we go for these get/set helpers, i'd probably split out the `Delete/RemoveFeature()` func as well to both separate the functionality, and also because "setting an empty string" does not really make me think of "removing a key/value pair from a map" (maybe the empty string is a value someone legitimately wants)

Reply:
I agree in principle, the reason why I did it that way was for consistency/since it might be confusing to have the features setters work differently than the plugin's –https://github.com/laurazard/cli/blob/b463a15f78e8c74c7f956ab5511d3c3ea60b1d57/cli/config/configfile/file.go#L345-L349

I'm happy to change this though.

Reply:
Oh, I didn't even notice that! No hard feelings here anyway